OneLogin.xcodeproj/project.pbxproj

```diff
@@ -508,7 +519,7 @@
 				DEVELOPMENT_TEAM = N8W395F695;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = "OneLogin-Info.plist";
-				INFOPLIST_KEY_CFBundleDisplayName = "One Login";
+				INFOPLIST_KEY_CFBundleDisplayName = "One Login - Debug";
```

Comment:
If it needs changing lets creating another ticket to make this change

Reply:
When it comes to actually setting these names, we should use the bundle variables instead of setting them manually. Eg:

`$(PRODUCT_NAME) - $(CONFIGURATION)` for non release builds
`$(PRODUCT_NAME)` for production build